src/pkg/backup/details/details.go

```diff
@@ -302,20 +302,22 @@ type ExchangeInfo struct {
 	Organizer   string    `json:"organizer,omitempty"`
 	ContactName string    `json:"contactName,omitempty"`
 	EventRecurs bool      `json:"eventRecurs,omitempty"`
+	Created     time.Time `json:"created,omitempty"`
+	Modified    time.Time `json:"modified,omitempty"`
 }
 
 // Headers returns the human-readable names of properties in an ExchangeInfo
 // for printing out to a terminal in a columnar display.
 func (i ExchangeInfo) Headers() []string {
 	switch i.ItemType {
 	case ExchangeEvent:
-		return []string{"Organizer", "Subject", "Starts", "Ends", "Recurring"}
+		return []string{"Organizer", "Subject", "Starts", "Ends", "Recurring", "Created", "Modified"}
```

Comment:
Long term question: if/as the list of info for each data type continues to grow we'll be more likely to overextend the terminal line width and have a fairly ugly output.  Let's keep these values in for now, but also keep in mind that we might want to reduce the display later on.